README.md

```diff
@@ -0,0 +1,47 @@
+<h1 align="center">
+    High-quality, ubiquitous, and portable telemetry to
+    enable effective observability
+</h1>
+
+<p align="center">
+  <a href="https://opentelemetry.io/">
+    <img src="https://opentelemetry.io/img/logos/opentelemetry-horizontal-color.svg" alt="OpenTelemetry logo" height="180">
+  </a>
+</p>
+
+<p align="center">
+OpenTelemetry is a collection of tools, APIs, and SDKs. Use it to
```

Comment:
```suggestion
OpenTelemetry is a collection of APIs, SDKs, and tools. Use it to
```

Reply:
There is an order of importance.

Reply:
good point, I copied that sentence from opentelemetry.io, so looks like we should change it there as well?